doozer/doozerlib/cli/scan_osh.py

```diff
@@ -1,29 +1,336 @@
 import click
 import koji
 import asyncio
+
+import requests
+import yaml
 from doozerlib.cli import cli, click_coroutine, pass_runtime
 from doozerlib.runtime import Runtime
-from doozerlib.exectools import fire_and_forget, cmd_gather_async, limit_concurrency
+from doozerlib.exectools import fire_and_forget, cmd_gather_async, limit_concurrency, cmd_gather
 from doozerlib.util import cprint
+from doozerlib.image import ImageMetadata
 from typing import Optional
+from jira import JIRA, Issue
+from tenacity import retry, stop_after_attempt, wait_fixed
+from doozerlib.rpm_utils import parse_nvr
+
+SCAN_RESULTS_URL_TEMPLATE = "https://cov01.lab.eng.brq2.redhat.com/osh/task/{task_id}/log/{nvr}/scan-results-imp.js" \
+                            "?format=raw"
+
+MESSAGE = """
+One or more vulnerabilities have been detected for the package *{package_name}*
+
+*Build*: [{nvr}|https://brewweb.engineering.redhat.com/brew/buildinfo?buildID={brew_build_id}]
+*Scan result*: https://cov01.lab.eng.brq2.redhat.com/osh/task/{scan_id}
+*View vulnerabilities*: https://cov01.lab.eng.brq2.redhat.com/osh/task/{scan_id}/log/{nvr}/scan-results-imp.js (Check 'defects')
+
+Once these issues are addressed, the Bug can be closed. But a new one will be opened if new vulnerabilities are found in the next build.
+
++POCs+
+*ProdSec*: If you have any questions regarding the vulnerability listed, email secalert@redhat.com
```

Comment:
I think we can safely replace this whole section now with a link to the policy document. Something like:

For information on triaging these issues and dispositioning this ticket, please review the [policy document for SAST Scanning Jira Tickets](https://docs.google.com/document/d/1GsT3tvw5qf3sOSAkTSVWnMIO6vzIuirhsVhhsbK_qf0/edit).